util/click_util.py

```diff
@@ -81,10 +81,11 @@ def convert(self, value, param, ctx):
     "tolerance_file_name": r"the name of the file containing the tolerances "
     + r"(per time step and variable)",
     "stats_file_name": r"the name of the stats file. No absolute path here, it will "
-    + r"always be create in the (perturbed) model_output_dir.",
-    "member_ids": r"member_id of the ensemble. Each member_id will generate a new set "
-    + r"of input files (comma separated list) these also serve as ID for input/output "
-    + r"directories, a seed is generate by the hash of the member_id",
+    + r"always be created in the (perturbed) model_output_dir.",
+    "member_num": r"number of ensemble members",
```

Comment:
```suggestion
    "member_num": r"number of ensemble members or a comma separated list of members (e.g. \"1,3,14\")",
```
Would this be the correct new description of this parameter?

Reply:
yes, that would be correct. One can also give the input simply as `--member-num 6,2,8` but `--member-num "6,2,8"` works equally well.